WordPress/src/main/java/org/wordpress/android/WordPress.java

```diff
@@ -666,6 +670,34 @@ public static String getUserAgent() {
         return mUserAgent;
     }
 
+    /*
+     * Since Android P:
+     * "Apps can no longer share a single WebView data directory across processes.
+     * If your app has more than one process using WebView, CookieManager, or any other API in the android.webkit
+     * package, your app will crash when the second process calls a WebView method."
+     *
+     * (see https://developer.android.com/about/versions/pie/android-9.0-migration)
+     *
+     * Also here: https://developer.android.com/about/versions/pie/android-9.0-changes-28#web-data-dirs
+     *
+     * "If your app must use instances of WebView in more than one process, you must assign a unique data
+     * directory suffix for each process, using the WebView.setDataDirectorySuffix() method, before
+     * using a given instance of WebView in that process."
+     *
+     * While we don't explicitly use a different process other than the default, making the directory suffix be
+     * the actual process name will ensure there's one directory per process, should the Application's
+     * onCreate() method be called from a different process any time.
+     *
+    */
+    private void setWebViewDataDirectorySuffixOnAndroidP() {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.P) {
+            String procName = getProcessName();
+            if (!TextUtils.isEmpty(procName)) {
+                WebView.setDataDirectorySuffix(procName);
```

Comment:
I thought of that given that's what is being done in https://github.com/wordpress-mobile/WordPress-Android/pull/10613/files#diff-17dbba974e19210536213a313661d5a7L628, but this crash in particular is too wide to just eat it and I'm afraid it will be only delaying the crash for as soon as any access to methods `android.webkit` package happens, so in essence it wouldn't fix it but rather only delay it for the next access (and we wouldn't get the user agent as well).
Thanks for the review!  🙇